src/main/java/uk/gov/hmcts/befta/AuthenticationRetryConfiguration.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.befta;
+
+import org.apache.commons.lang3.math.NumberUtils;
+
+public class AuthenticationRetryConfiguration {
+    private int retryAttempts;
+    private int retryMaxTimeInSeconds;
+    private int retryMultiplierInMilliseconds;
+
+    public AuthenticationRetryConfiguration(String retryAttempts,
+                                            String retryMaxTimeInSeconds,
+                                            String retryMultiplierInMilliseconds) {
+        this.retryAttempts = NumberUtils.toInt(retryAttempts);
+        this.retryMaxTimeInSeconds = NumberUtils.toInt(retryMaxTimeInSeconds);
+        this.retryMultiplierInMilliseconds = NumberUtils.toInt(retryMultiplierInMilliseconds);
```

Comment:
pls accept int values to constructor and centralise conversion of String configs into integer values inside the `TestAutomationConfig` class.

Reply:
changed the values to int and updated TestAutomationConfig class accordingly